envoy/config/filter/http/rate_limit/v2/BUILD

```diff
@@ -0,0 +1,9 @@
+load("//bazel:api_build_system.bzl", "api_proto_library")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_library(
+    name = "rate_limit",
+    srcs = ["rate_limit.proto"],
+    visibility = ["//docs"],
```

Comment:
I think we can make this less verbose by just always adding `//doc` in `api_proto_library` (it can't ever hurt), and making `visibility` optional and by default private.

Reply:
Cool idea! I made it private by default, and add //docs always (thereby switching from private to `["//docs"]` since private is not combinable).